### PR TITLE
Aggregate metric series

### PR DESCRIPTION
**Pull request by mgrauer**

### CODE REVIEW

File: core/controllers/components/UtilityComponent.php [hunk 1]
```diff
@@ -954,4 +954,50 @@ public static function setCookie($request, $name, $value, $expires)
...
+
+
+        /** @var int $len1 */
+        $len1 = strlen($string1);
+        /** @var int $len2 */
+        $len2 = strlen($string2);
+        /** @var int $commonSuffixInd */
+        $commonSuffixInd = $len1;
+        /** @var int $i */
+        $i = 1;
```
Comment (cpatrick) [inline core/controllers/components/UtilityComponent.php]: Why is `$i` not in the loop?

**Comment by cpatrick** [comment]
LGTM other than the nit and as long as StyleCI and Travis get happy.

File: core/controllers/components/UtilityComponent.php (hunk 1 above)
Comment (mgrauer) [inline core/controllers/components/UtilityComponent.php]: Vestigal I'm guessing.  Can't even remember.

**Comment by mgrauer** [comment]
It's fitting that on the eve of dropping support for pgsql and sqlite in the tracker I had to deal with a db compatibility issue.

**Comment by cpatrick** [comment]
:panda_face:

**Comment by mgrauer** [comment]
@cpatrick 

PT one last look

**Comment by cpatrick** [comment]
lgtm.

**Pull request merged**